components/clp-package-utils/clp_package_utils/general.py

```
@@ -510,30 +509,11 @@ def validate_worker_config(clp_config: CLPConfig):
-    clp_config: CLPConfig, logs_dir: pathlib.Path, settings_json_path: pathlib.Path
-):
-    if not settings_json_path.exists():
-        raise ValueError(
-            f"{WEBUI_COMPONENT_NAME} {settings_json_path} is not a valid path to Meteor settings.json"
-        )
-
-    try:
-        validate_path_could_be_dir(logs_dir)
-    except ValueError as ex:
-        raise ValueError(f"{WEBUI_COMPONENT_NAME} logs directory is invalid: {ex}")
+def validate_webui_config(clp_config: CLPConfig,
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)